pytket-examples-book/requirements.txt

```diff
@@ -0,0 +1,8 @@
+networkx == 2.8.8
```

Comment:
The reason that this is different is that I require some additional dependencies to execute the qujax, UCC_vqe and creating_backends examples. I think its useful to have these in executed form.

I think it would also be useful to have separate requirements files for the manual and the examples. They do use different packages. What do you think?

Reply:
I have separated the dependencies into two separate files

`manual_requirements.txt`
and
`pytket-examples-book/example_requirements.txt`

let me know if this is acceptable.